src/vosk_api.cc

```diff
@@ -129,12 +129,12 @@ void vosk_recognizer_set_grm(VoskRecognizer *recognizer, char const *grammar)
     ((Recognizer *)recognizer)->SetGrm(grammar);
 }
 
-void vosk_recognizer_set_ep_mode(VoskRecognizer *recognizer, VoskEpMode mode)
+void vosk_recognizer_set_silence(VoskRecognizer *recognizer, float length)
```

Comment:
scale, not length, right

Reply:
I've changed it to "seconds" because it's the duration of silence in seconds.